util/rlp/fuzz/std/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "rlp-std"
```

Comment:
what's the purpose of this crate? we will add the given test case as a unit test when we prepare the fix.